src/test/java/net/sf/jabref/logic/util/io/UtilFindFileTest.java

```diff
@@ -1,45 +1,64 @@
 package net.sf.jabref.logic.util.io;
 
 import java.io.File;
+import java.nio.file.Paths;
 import java.util.Arrays;
 import java.util.Collection;
 import java.util.Collections;
 import java.util.List;
 import java.util.Map;
+import java.util.Set;
 
 import net.sf.jabref.FileBasedTestCase;
 import net.sf.jabref.Globals;
 import net.sf.jabref.model.entry.BibEntry;
+import net.sf.jabref.support.DevEnvironment;
 
-import org.junit.Assert;
 import org.junit.Ignore;
 import org.junit.Test;
-
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assume.assumeFalse;
 /**
  * Testing Util.findFile for finding files based on regular expressions.
  *
  * @author Christopher Oezbek <oezi@oezi.de>
  */
 public class UtilFindFileTest extends FileBasedTestCase {
 
-
     @Test
-    @Ignore(value = "works on windows but not on linux")
     public void testFindAssociatedFiles() {
+
         Collection<BibEntry> entries = Collections.singletonList(entry);
         Collection<String> extensions = Arrays.asList("jpg", "pdf");
-        Collection<File> dirs = Arrays.asList(new File(root.getAbsoluteFile() + "/pdfs/"),
-                new File(root.getAbsoluteFile() + "/graphicsDir/"));
+        Collection<File> dirs = Arrays.asList(rootDir.resolve("graphicsDir").toFile(),
+                rootDir.resolve("pdfs").toFile());
 
         Map<BibEntry, List<File>> results = FileUtil.findAssociatedFiles(entries, extensions, dirs, Globals.prefs);
 
-        Assert.assertEquals(2, results.get(entry).size());
-        Assert.assertTrue(
-                results.get(entry).contains(new File(root.getAbsoluteFile() + "/graphicsDir/subDir/HipKro03test.jpg")));
-        Assert.assertFalse(
-                results.get(entry).contains(new File(root.getAbsoluteFile() + "/graphicsDir/subDir/HipKro03test.png")));
-        Assert.assertTrue(results.get(entry).contains(new File(root.getAbsoluteFile() + "/pdfs/sub/HipKro03-sub.pdf")));
+        assertEquals(2, results.get(entry).size());
+        assertTrue(results.get(entry)
+                .contains(rootDir.resolve(Paths.get("graphicsDir", "subDir", "HipKro03test.jpg")).toFile()));
```

Comment:
Yeah, that was previous  Error. Now the tests work on linux, too. But not on the CI Server.